tracking_service/src/main/java/com/tel_ran/tracking_service/entity/CustomerEntity.java

```diff
@@ -23,4 +23,14 @@
     @OneToMany
     List<ShipmentEntity> shipments;
 
+    public CustomerEntity(String firstName, String lastName) {
```

Comment:
If you define an entity, named 'CustomerEntity', your should either:
- name your table in the db as "CustomerEntity"
- annotate the entity class with @Table where you specify the real table name.